BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs

```diff
@@ -38,10 +38,9 @@ public void ToggleAirTapToCreateEnabled()
         private XRAnchorStore m_anchorStore = null;
         private Dictionary<TrackableId, string> m_incomingPersistedAnchors = new Dictionary<TrackableId, string>();
 
-        async void Start()
+        protected async void OnEnable()
```

Comment:
I am sure there is a reason for this change.. may I know why? I am a little concerned of this change especially with remoting scenarios. For basic sample we don't have any special handling for late xr initialization since we rely on Start() happening later than XR initialization. Given, OnEnable() is called a little before, I doubt if there would be enough time for xr to initialize before looking for anchor manager and subsystems.